similarity/src/main/java/experiments/Minio.java

```diff
@@ -0,0 +1,46 @@
+package experiments;
+
+import org.cogcomp.Datastore;
+
+import edu.illinois.cs.cogcomp.core.utilities.configuration.ResourceManager;
+
+public class Minio {
+	public void upload() throws Exception{
+		String file="config/datastore.properties";
+		ResourceManager rm=new ResourceManager(file);
+		String endpoint=rm.getString("ENDPOINT");
+		String accessKey=rm.getString("ACCESS-KEY");
+		String secretKey=rm.getString("SECRET-KEY");
+		Datastore ds = new Datastore(endpoint,accessKey,secretKey);
+		/**
+		ds.publishFile("org.cogcomp.wordembedding",
+                "word2vec.txt", 1.5,
+                "/shared/bronte/sling3/data/vectors-enwikitext_vivek200.txt",
```

Comment:
I removed this class. It is used to upload file to our server. 
I also removed the experiments folder. There is no need to have them in the package.